server/applications.json

```diff
@@ -656,8 +656,10 @@
                         "windows": [
                             "C:\\Program Files\\Side Effects Software\\Houdini 19.5.805\\bin\\houdini.exe"
                         ],
-                        "darwin": [],
-                        "linux": []
+                        "darwin": [
+                            "/Applications/Houdini/Houdini19.5.805/Frameworks/Houdini.framework/Versions/Current/Resources/bin/houdini"
```

Comment:
There are also examples online that seem to refer to:
```
/Applications/Houdini/Houdini20.0.506/Frameworks/Houdini.framework/Resources/bin/houdini
```

I'm not sure what the good default is for Mac - so this answer may need to come from an active mac user :)